### DISCUSSION TITLE
.CSV export reports

### DISCUSSION THREAD

**Discussion by lvschie** [crater-invoice-inc/crater#755] [Ideas]
I've started using Crater a few weeks ago, and love it so far. There are a couple of small features that are missing in my opinion, one of them being an export function of the reports. My accountant wants them in Excel or CSV format so he can easily do some checks and calculations. This is a procedure that is both quarterly and yearly and is quite time consuming when having to do these manually.

As you might guess, the idea/request is a button next to the "Download PDF" button in the reports screen for exporting the selected report to CSV. My guess is that this shouldn't have too much implications from a technical perspective.

Oh, and being able to enter a starting range for invoice- and quotation numbers would also be a very welcome addition! I use a different range each year, like many other companies do.

**Comment by JeroenSteen** [comment]
I'm also looking for this feature, because I want to merge the Crater-data with my own Excel-file.

**Comment by gsluthra** [comment]
Is this feature (_ability to export reports in csv format_) on crater team's roadmap this year? Many folks will want this feature.
